pkg/koordlet/metricsadvisor/devices/gpu/collector_gpu_linux.go

```diff
@@ -181,7 +179,8 @@ func (g *gpuDeviceManager) getNodeGPUUsage() []metriccache.MetricSample {
 	return gpuMetrics
 }
 
-func (g *gpuDeviceManager) getTotalGPUUsageOfPIDs(pids []uint32) []metriccache.GPUMetric {
```

Comment:
we can pass container/pod id, and build metric meta inside
`getPodOrContainerGPUUsage(id, pids)`